### PR TITLE
docs(fast-dna): remove reference to Angular

### PR DESCRIPTION
**Pull request by eyelidlessness**
<body>My friends Nick and Jane say this project doesn’t actually support Angular.

<footer>I like to help OSS projects maintain their documentation.

### CODE REVIEW

**Comment by chrisdholt** [comment]
Hey @eyelidlessness thanks for pushing up a PR! Can you update your merge message to align with the guidance in the [Wiki](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance)?

I agree that our primary focus in the repo has been on React for many of our packages, specifically revolving around components and tooling; however, many of them are framework agnostic and can still be used to accelerate Angular projects. With that said, I think this is good feedback and worth taking a look at.

**Comment by eyelidlessness** [comment]
Hello @chrisdholt I believe I’ve updated my PR to match the standards you linked to. Interestingly the HTML body and footer tags make no rendering difference for me.

At any rate Nick and Jane laughed when I glanced at the project and asked if it really supports Angular. I can’t speak to the actual level of support but I was afraid that the walk up experience might misrepresent the focus of the project.

**Comment by nicholasrice** [comment]
I can see how this messaging could be misleading. I generally don't describe the project as having direct Angular support because we have only completed the end to end component system in React and the project have very little Angular-specific code. This is a mono-repo readme though, which is speaking to the larger eco-system of packages - some of which could be used in a Angular project if desired.

Perhaps we could change this sentence to: "Many of the resources are written to be framework agnostic and will work with any component library and JavaScript framework (React, Angular, etc.). We also support a number of React-specific packages for UI generation."

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # FAST-DNA overview
 FAST-DNA is tooling, resources, and components used to accelerate building Web sites and applications.
 
-Teams spend significant time building parts of the design system (colors, variable font, typography, grid, components, etc.) which can slow adoption of the latest design guidelines and related technologies. FAST-DNA solves this problem by providing tooling and resources to accelerate development with code that is performant and accessible. The resources work with multiple component frameworks such as (Fabric, Foundation, etc.) as well as JavaScript frameworks and libraries (React, Angular, etc.).
+Teams spend significant time building parts of the design system (colors, variable font, typography, grid, components, etc.) which can slow adoption of the latest design guidelines and related technologies. FAST-DNA solves this problem by providing tooling and resources to accelerate development with code that is performant and accessible. The resources work with multiple component frameworks such as (Fabric, Foundation, etc.) as well as JavaScript frameworks and libraries (React, etc.).
```
Comment (nicholasrice) [inline README.md]: I think that whole sentence could be adjusted to paint a clearer picture. I think the following sentence is more illustrative of the current project:

"In addition to a number of React packages, many of these resources are written to be framework agnostic and will work with any component library (Fabric, Foundation, etc.) and JavaScript framework (React, Angular, etc.)."

**Comment by eyelidlessness** [comment]
Thanks for the feedback Nick. I’ve updated the pr with the language you suggested.

**Comment by chrisdholt** [comment]
Looks like another 9 commits just got added? I think these were already merged into master.

**Comment by eyelidlessness** [comment]
Sorry I had git mishaps, not used to dealing with forks. I think it's fixed now?

**Pull request merged**